OPHD/Things/Structures/Structure.cpp

```diff
@@ -310,8 +310,5 @@ void Structure::die()
 {
 	Thing::die();
 
-	#if defined(_DEBUG)
-	std::cout << "Holy shit, a Structure died!!!" << std::endl;
```

Comment:
Always the professional huh? :D

Reply:
Well, I did leave in "He's dead Jim!" :wink: